project/VersionedGhPages.scala

```diff
@@ -10,34 +10,35 @@ import com.typesafe.sbt.git.GitRunner
 // Copied from sbt-ghpages to avoid blowing away the old API
 // https://github.com/sbt/sbt-ghpages/issues/10
 object VersionedGhPages {
-	def cleanSiteForRealz(dir: File, git: GitRunner, s: TaskStreams, apiVersion: (Int, Int)): Unit ={
-		val toClean = IO.listFiles(dir).collect {
-			case f if f.getName == "api" => new java.io.File(f, s"${apiVersion._1}.${apiVersion._2}")
-			case f if f.getName != ".git" && f.getName != "CNAME" => f
-		}.map(_.getAbsolutePath).toList
-		if (!toClean.isEmpty)
-			git(("rm" :: "-r" :: "-f" :: "--ignore-unmatch" :: toClean) :_*)(dir, s.log)
-		()
-	}
+  def cleanSiteForRealz(dir: File, git: GitRunner, s: TaskStreams, apiVersion: (Int, Int)): Unit = {
```

Comment:
The latest sbt-ghpages could make all this nonsense go away, but that's probably out of scope for this request.